src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,225 @@
+                }
+
+        } catch (IOException e) {
+            System.out.println(TEXT_SPACER + "No save file. New file will be created");
+        } catch (DukeExceptions e){
+            System.out.println(e.getMessage());
+        }
+        return numOfTasks;
+    }
+
+    public static void saveToFile(ArrayList userList, int numTask) throws IOException {
+        FileWriter updateFile = new FileWriter(path + "/task.txt");
+        //TO FIX: DOESN'T UPDATE DONE STATUS
+        for(int i = 0; i < numTask; i++) {
+            String stringToSave = ((Task)userList.get(i)).getTaskType() + " | " + getDoneStatus(userList, i) +
+                    " | " + ((Task)userList.get(i)).getDescription() + "\n";
+            updateFile.write(stringToSave);
+        }
+        updateFile.close();
+    }
+
+    private static String getDoneStatus(ArrayList userList, int numTask) {
+        if (((Task)userList.get(numTask)).getStatusIcon().equals("[X]")){
+            return "1";
+        }
+        return "0";
+    }
+
+    private static void printWelcome() {
+        String logo = " ____        _        \n"
+                + "|  _ \\ _   _| | _____ \n"
+                + "| | | | | | | |/ / _ \\\n"
+                + "| |_| | |_| |   <  __/\n"
+                + "|____/ \\__,_|_|\\_\\___|\n";
+        System.out.println("Hello from\n" + logo);
+    }
+
+    private static void printList(ArrayList userLists, int numOfTasks) {
+        if (numOfTasks == 0) {
+            System.out.println(TEXT_SPACER + "List is empty!");
+        } else {
+            System.out.println(TEXT_SPACER + "Here's your list of tasks:");
+            for (int i = 0; i < numOfTasks; i++) {
+                System.out.println("    " + (i + 1) + "." + userLists.get(i).toString());
+            }
+        }
+    }
+
+    private static void completeTask(ArrayList userLists, String splitInput) {
+        int chosenEntry;
+        chosenEntry = Integer.parseInt(splitInput) - 1;
+        ((Task)userLists.get(chosenEntry)).markAsDone();
+
+        System.out.println(TEXT_SPACER + "Good job on completing a task!");
+        System.out.println(TEXT_SPACER + userLists.get(chosenEntry).toString());
+    }
+
+    private static void storeTasks(ArrayList userLists, String userLineInput,
+                                   int numOfTasks, String userCommand) throws DukeExceptions{
+        if (userCommand.equals("event")){
+            storeEvent(userLists, userLineInput, numOfTasks);
+        } else if(userCommand.equals("deadline")) {
+            storeDeadline(userLists, userLineInput, numOfTasks);
+        } else {
+            storeToDo(userLists, userLineInput, numOfTasks);
+        }
+        System.out.println(TEXT_SPACER + "Aight, I've added the following task to your list:");
+        System.out.println(TEXT_SPACER + userLists.get(numOfTasks).toString());
+        System.out.println(TEXT_SPACER + "Now you have " + (numOfTasks + 1) + " tasks in your list");
```

Comment:
You have `storeEvent()`, `storeDeadline()`, `storeToDo()`. Perhaps abstract the `println` statements to achieve [SLAP](https://nus-cs2113-ay2122s1.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard)?